src/content/components/ProfileThreadHeaderBar.js

```diff
@@ -72,9 +74,11 @@ class ProfileThreadHeaderBar extends PureComponent {
   render() {
     const {
       thread, interval, rangeStart, rangeEnd, funcStackInfo, selectedFuncStack,
-      isSelected, style, threadName, processDetails,
+      isSelected, style, threadName, processDetails, isHidden,
     } = this.props;
-
+    if (isHidden) {
+      return <li className='profileThreadHeaderBarHidden' />;
+    }
```

Comment:
This is so that Reorderable can just work without any additional complex trickery. I added comments to be more clear why they are there.